src/istio/mixerclient/report_batch.cc

```diff
@@ -71,12 +72,34 @@ void ReportBatch::FlushWithLock() {
 
   ++total_remote_report_calls_;
   auto request = batch_compressor_->Finish();
-  ReportResponse* response = new ReportResponse;
+  std::shared_ptr<ReportResponse> response{new ReportResponse()};
```

Comment:
note to reviewer, i slipped this in while I was at it.  let me know if you want me to revert

Reply:
I am fine with this change. Thanks!